ui/hooks/src/moderation-requests.ts

```diff
@@ -108,6 +108,13 @@ export interface ModeratedItemsReponse extends PaginatedResponse {
   results: IModeratedItem[];
 }
 
+/**
+ * Creates moderation entry
+ * @param url - the request url
+ * @param data - moderation data specifying content id, type, signed data and signature
```

Comment:
don't list the properties, they will be extracted from the interface.
It makes it hard to maintain, if you replace/rename a property this doc will be obsolete